meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi

```diff
@@ -30,14 +30,17 @@
 			linux,default-trigger = "heartbeat";
 		};
 	};
+        chosen {
```

Comment:
Can you replace space with tab in this block? The rest of the file is using tab instead of spaces.